python-stdlib/os/os/__init__.py

```diff
@@ -1,8 +1,40 @@
-# Replace built-in os module.
-from uos import *
+# Include built-in os module.
+import sys
+
+__path = sys.path
+try:
+    sys.path.clear()
+    from os import *
+finally:
+    sys.path.extend(__path)
```

Comment:
Hey @jimmo is this what you had in mind as the new / current best practice for importing and extending builtins here?